code/modules/client/client_procs.dm

```diff
@@ -421,7 +424,7 @@ GLOBAL_LIST_INIT(whitelisted_client_procs, list(
 	apply_clickcatcher()
 
 	if(prefs.lastchangelog != GLOB.changelog_hash) //bolds the changelog button on the interface so we know there are updates.
-		winset(src, "infowindow.changelog", "background-color=#ED9F9B;font-style=bold")
+		stat_panel.send_message("changelog_read", FALSE)
```

Comment:
```suggestion
		stat_panel.send_message("changelog_read", FALSE)
```
I don't really like big bold buttons that have to be pressed to go away personally and other people with adhd or otherwise might find it annoying as well, either remove it or make it a preference imo